sdk/formrecognizer/tests.yml

```diff
@@ -8,5 +8,21 @@ jobs:
         - name: azure-ai-formrecognizer
           groupId: com.azure
           safeName: azureaiformrecognizer
+      # resource creation is disabled because we are trying to test the stable endpoint.	
+      # Will enable it when service rolls out AAD to all region.	
+      DisableAzureResourceCreation: true
```

Comment:
Didn't know about this, cool

Reply:
Hehe...Thanks to @mssfang and TA to have done this before 🚆 .